sunpy/data/data_manager/cache.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, downloader, storage, cache_dir, expiry=10*u.day):
         self._cache_dir = Path(cache_dir)
         self._expiry = expiry if expiry is None else TimeDelta(expiry)
 
-    def download(self, urls, redownload=False):
+    def download(self, urls, namespace='sunpy.', redownload=False):
```

Comment:
I think if we can avoid adding the "." to the namespace, that would be nice.

Reply:
Why does this have to be hardcoded as `sunpy`? Isn't this already sorted out by `_get_module`?

Reply:
I added this hardcoded default because a few places in sunpy's codebase were using `cache.download()` directly, instead of going through `manager.require()`, which is what provides `_get_module`.

Reply:
I am not convinced we need to do this namespacing for direct invocations of `cache.download()`. The cache is designed to be able to be used by user code or by libraries, whereas the manager is really a library interface.

Reply:
I see. How about we pass an empty string as the default argument?

Reply:
`None` would be marginally cleaner than `""` although either works.

Reply:
I'll go with `''` because I'll have to introduce a check with `None`, whereas concatenation will happen with `''` in a concise way.